### PR TITLE
Upgrade master to Node 8.9.2

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

Upgrading to Cardano Node 8.9.2 also requires updating Ogmios to 6+.
Ogmios 6+ is breaking wrt previous versions, but the work to address the breaking changes was already done on the `conway-era` branch.

# Proposed Solution
1. ~~Branch from `conway-era`: `chore/LW-7691-conway-era-variant-for-master-merge`~~ Branched from `master` into `chore/LW-7691-master-merge`. Merged `comway-era` into it.
2. Update to use stable DbSync (13.2.0.2) instead of the `sanchonet-*` ones.
3. Adapt to schema changes and stabilise.
4. ~~Merge to `master` ONLY AFTER db-sync migration for the existing deployments is planned and ready to execute.~~ not needed

### CODE REVIEW

File: compose/aarch64.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 services:
   cardano-db-sync:
-    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-13.1.0.0}
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-13.2.0.2}
```
Comment (michalrus) [inline compose/aarch64.yml:5]: These ARM images don't exist yet, I will create them shortly.

Comment (michalrus) [reply compose/aarch64.yml:5]: Now they do.

File: packages/e2e/local-network/Dockerfile [hunk 2]
```diff
@@ -5,28 +5,26 @@ FROM ubuntu:${UBUNTU_VERSION} as builder
 ENV DEBIAN_FRONTEND=nonintercative
 
 WORKDIR /build
-ARG CARDANO_NODE_BUILD_URL=https://update-cardano-mainnet.iohk.io/cardano-node-releases/cardano-node-1.35.4-linux.tar.gz
-ARG CARDANO_NODE_BUILD_URL_ARM64=https://github.com/input-output-hk/ogmios-tracker/releases/download/0.1.0/cardano-node-1.35.4-aarch64-linux.tar.gz
+ARG CARDANO_NODE_BUILD_URL=https://github.com/IntersectMBO/cardano-node/releases/download/8.9.2/cardano-node-8.9.2-linux.tar.gz
+ARG CARDANO_NODE_BUILD_URL_ARM64=https://github.com/input-output-hk/ogmios-tracker/releases/download/0.1.0/cardano-node-8.9.2-aarch64-linux.tar.gz
```
Comment (michalrus) [inline packages/e2e/local-network/Dockerfile:9]: This doesn't exist either ⏳

Comment (michalrus) [reply packages/e2e/local-network/Dockerfile:9]: It does now.

**Comment by mirceahasegan** [comment]
Moving back to 'DRAFT' state until changes from [PR-1222](https://github.com/input-output-hk/cardano-js-sdk/pull/1222) are integrated here too

**Comment by mirceahasegan** [comment]
This PR will remain in DRAFT state until the changes are thoroughly tested.

File: nix/cardano-services/deployments/default.nix [hunk 3]
```diff
@@ -225,55 +225,23 @@ in
...
-              (map (v: "/v${v}/chain-history") versions.chainHistory)
-              (map (v: "/v${v}/network-info") versions.networkInfo)
-              (map (v: "/v${v}/rewards") versions.rewards)
-              (map (v: "/v${v}/tx-submit") versions.txSubmit)
-              (map (v: "/v${v}/utxo") versions.utxo)
-              (map (v: "/v${v}/handle") versions.handle)
-              (map (v: "/v${v}/provider-server") versions.stakePool)
-              (map (v: "/v${v}/stake-pool-provider-server") versions.stakePool)
-            ];
+        "dev-sanchonet@us-east-1@v1" = final: {
```
Comment (mirceahasegan) [inline nix/cardano-services/deployments/default.nix]: @bernokl , can you take a look at these changes from the merge?
I am unsure wether they will mess any of the deployment work.

File: packages/ogmios/test/ogmiosToCore/__snapshots__/block.test.ts.snap [hunk 4]
```diff
@@ -288,7 +288,7 @@ Object {
               "cbor": "d87980",
               "constructor": 0n,
               "fields": Object {
-                "cbor": "9fff",
```
Comment (mirceahasegan) [inline packages/ogmios/test/ogmiosToCore/__snapshots__/block.test.ts.snap]: @AngelCastilloB . Please review this change. I had to update this as it was failing after the CBOR list fix. I assumed this CBOR was also wrong.

Comment (AngelCastilloB) [reply packages/ogmios/test/ogmiosToCore/__snapshots__/block.test.ts.snap]: This change is correct @mirceahasegan ✅

File: packages/e2e/test/wallet_epoch_0/PersonalWallet/phase2validation.test.ts [hunk 5]
```diff
@@ -70,7 +70,10 @@ describe('PersonalWallet/phase2validation', () => {
     wallet.shutdown();
   });
 
-  it('can detect a phase2 validation failure and emit the transaction as failed', async () => {
+  // The submit fails instead of being able to send it.
+  // TODO: re-enable this after understanding how to force submit API to submit a transaction that
+  // will fain in phase2 validation
+  it.skip('can detect a phase2 validation failure and emit the transaction as failed', async () => {
```
Comment (mkazlauskas) [inline packages/e2e/test/wallet_epoch_0/PersonalWallet/phase2validation.test.ts:76]: (not a change request for this PR) do we need to use tx submit api in local network?

File: packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/ChainHistoryBuilder.test.ts [hunk 6]
```diff
@@ -225,4 +225,60 @@ describe('ChainHistoryBuilder', () => {
       expect(result.size).toEqual(0);
     });
   });
+
+  // TODO: enable after conway era hardfork
+  describe.skip('queryProposalProceduresByIds', () => {
```
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/ChainHistoryBuilder.test.ts:230]: No. It's at babbage : https://github.com/input-output-hk/cardano-js-sdk/pull/1251/files#diff-a141cc935ff48eaadd336d5ce8fca75248dbfb58efa7cb22339ee38a84960a69R160

File: packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/mappers.test.ts [hunk 7]
```diff
@@ -263,19 +266,21 @@ describe('chain history mappers', () => {
         ...stakeCert,
         registration: false
       } as WithCertType<StakeCertModel>);
-      expect(registrationResult).toEqual<WithCertIndex<Cardano.StakeAddressCertificate>>({
-        __typename: Cardano.CertificateType.StakeRegistration,
+      expect(registrationResult).toEqual<WithCertIndex<Cardano.NewStakeAddressCertificate>>({
+        __typename: Cardano.CertificateType.Registration,
         cert_index: 0,
+        deposit: 0n,
```
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/mappers.test.ts:272]: > Maybe we need to re-generate the test snapshot
I regenerated it (https://github.com/input-output-hk/cardano-js-sdk/pull/1251/commits/cc14ab9e97c9b7b66fc4fa85c85e8cde33d3308b)

File: packages/e2e/test/wallet_epoch_0/PersonalWallet/phase2validation.test.ts (hunk 5 above)
Comment (mirceahasegan) [reply packages/e2e/test/wallet_epoch_0/PersonalWallet/phase2validation.test.ts:76]: I will add a task to re-enable it after the merge.

**Review by mkazlauskas — APPROVED** [review]
Great work everyone involved, huge effort! :muscle: :heart:

**Review by AngelCastilloB — DISMISSED** [review]
Amazing work guys! great work! 🥇

**Review by mkazlauskas — DISMISSED** [review]

**Review by mkazlauskas — APPROVED** [review]

**Review by pczeglik-iohk — APPROVED** [review]

**Pull request merged**